worldedit-core/src/main/java/com/sk89q/worldedit/internal/expression/Expression.java

```diff
@@ -69,6 +78,22 @@
 public class Expression {
 
     private static final ThreadLocal<Stack<Expression>> instance = new ThreadLocal<>();
+    private static final ExecutorService evalThread = Executors.newCachedThreadPool(
+            new ThreadFactoryBuilder()
+                    .setDaemon(true)
+                    .setNameFormat("worldedit-expression-eval-%d")
+                    .build());
+
+    // A workaround for testing -- we can't easily create a LocalConfiguration
```

Comment:
@me4502 Want to explain why `LocalConfiguration` can't be created here -- it uses `BlockType#getId`, which relies on the underlying `Platform` to provide registries, etc. But this test is in `-core`, where we have no `Platform` implementation. I think the best solution is to remove references to `BlockType` -- we should just be writing the string IDs there now. I'm not sure if that change should be in this PR as well though.

Reply:
Hmm, I'll take a look into uncoupling BlockType#getId() from the Platform, it doesn't overly make sense for it to depend on it.